Topics/Development_Process/Technical_Documents/Intro_to_rfcs.md

```diff
@@ -0,0 +1,96 @@
+- An RFC can be understood as “Here is the direction we want to go, are we missing anything?”
+- There are 4 general phases in an RFC life cycle
+  - Proposal stage: the RFC is drafted
+  - Commenting stage: feedback and discussion ensue
+  - Decision stage: ideally, the reviewers and the author align on the path forward
+  - Archive stage: the RFC is archived for future reference
+
+## **Why are RFCs useful?**
+
+- Mitigates the risk of the decision made
+  - RFCs act as a kind of code review but at the planning stage
+  - Problems or considerations can be surfaced before resources have been spent
+- Facilitates better technical decisions and collaboration
+  - Building software is a social exercise
+  - RFCs facilitates communication and decision-making between groups
+- Allows for individual contribution and responsibility on all levels
+  - When done right, individuals of all levels in the organization are empowered and encouraged to make impactful decisions while seeking input from others
+- Creates documentation for future context
+  - An archive of past RFCs act as a record of previous discussions and decisions
+- Is an asynchronous process
+  - Multiple RFCs can run in parallel
+  - Increases the flexibility of collaborative discussions
+
+## **When to RFC?**
+
+If it crosses your mind that an RFC might be required to make a change or implement a design, it probably is a good idea to make one.
+
+Additionally, an RFC is generally a good idea when:
+
+- You are building something from scratch (new endpoint, component, system, library, application, etc.).
+- You will impact more than one system or other team members.
+- You would like to define a contract of interface between clients or systems.
+- You are adding a new dependency.
+- You are adding or replacing languages or tools to the stack.
+- You are in doubt of whether to write one.
+
+## **How to RFC?**
+
+RFCs are a guideline, not a rigid methodology. They are usually tailored to the specific needs of teams and organizations, so multiple variations exist both in the actual document and the surrounding process.
+
+### **Adopting RFCs in a team**
+
+Some considerations when adopting RFCs in a team:
+
+- Success using RFCs isn’t guaranteed – when adopting it for the first time in a team, think of it initially as a collective experiment and reflect on whether:
+  - Has using RFCs led to better decisions?
+  - Is your team better informed and making decisions faster?
+  - Are different groups equally participating in comments?
+  - Can the process be lighter while achieving similar outcomes?
+- It is usually a good idea to define a time-limit on the commenting stage to avoid it lasting forever.
+- The tool you use can vary – some options include collaborative docs (ex. Google Docs, Microsoft Word, etc.), markdown doc in a git repo, a slide presentation, etc.
+
+### **Getting started: RFC templates**
```

Comment:
maybe you could add a brief explanation of why templates are essential and how different teams can customize them based on their own needs.